package.json

```
@@ -13,22 +13,22 @@
     "./use-at-your-own-risk": "./lib/unsupported-api.js"
   },
   "scripts": {
+    "build:docs:update-links": "node tools/fetch-docs-links.js",
+    "build:release": "node Makefile.js generateRelease",
+    "build:release:alpha": "node Makefile.js generatePrerelease -- alpha",
+    "build:release:beta": "node Makefile.js generatePrerelease -- beta",
+    "build:release:publish": "node Makefile.js publishRelease",
+    "build:release:rc": "node Makefile.js generatePrerelease -- rc",
+    "build:site": "node Makefile.js gensite",
+    "build:webpack": "node Makefile.js webpack",
+    "lint": "node Makefile.js lint",
+    "lint:fix": "node Makefile.js lint -- fix",
+    "lint:docs:js": "node Makefile.js lintDocsJS",
+    "lint:docs:js:fix": "node Makefile.js lintDocsJS -- fix",
```

```suggestion
    "lint:fix:docs:js": "node Makefile.js lintDocsJS -- fix",
```

Per https://eslint.org/docs/latest/developer-guide/package-json-conventions#modifiers, `:fix` should be before the target (`docs:js`)?

---

Yes, that’s where it should be per the grammar at the top. Good catch.